flyte.config

```diff
@@ -5,6 +5,6 @@ workflow_packages=myapp.workflows
 [auth]
 # Uncomment if you want to use a service account for all your tasks and workflow executions. This service account should be created by you and available on the k8s cluster and it will be used to read and write data from the backend store like S3/GCS, or to connect to any services that you use in your tasks.
 # to the blobstore (e.g. s3) used to write task execution outputs.
-# kubernetes-service-account=demo
+# kubernetes_service_account=demo
 # You can set this prefix to specify where task output schema and blobs should be written to.
-raw_output_data_prefix=s3://my-s3-bucket/cookbook
+raw_output_data_prefix=s3://flyte-demo/raw_data
```

Comment:
did this bucket change? i thought we were still using my-s3-bucket https://github.com/flyteorg/flyte/pull/905/files